### PR TITLE
Add the raw uuid to the response.

### PR DESCRIPTION
**Pull request by chrislentz**

### CODE REVIEW

**Comment by luisfarzati** [comment]
Thank you @chrislentz , looks good, I only have one comment that I'll leave in the right section.

File: src/captureScreenshot.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ exports.handler = async (event, callback) => {
     return buffer;
   });
 
-  const url = await S3Bucket.upload(buffer, imageType, options.s3key);
+  const response = await S3Bucket.upload(buffer, imageType, options.s3key);
 
-  return { url };
+  return JSON.stringify(response);
```
Comment (luisfarzati) [inline src/captureScreenshot.js]: Not sure why you wanted to return a string instead of the object?

Comment (chrislentz) [reply src/captureScreenshot.js]: This was an artifact from another PR I have coming shortly to support private API Gateway request/response, but yes if not a response back to API Gateway, it should be an object. Good Catch!

**Comment by luisfarzati** [comment]
Excellent, thanks! And yes, maybe in the same way there's a `normalizedEvent` function, there could be a `normalizedResponse` or something like that. Otherwise right now the input is compatible but the response is not 🤦‍♂️

**Pull request merged**